docs/pages/context.rst

```diff
@@ -631,7 +631,7 @@ Here's an example that might give you a better understanding of how
   from returns.io import IOResultE
   from returns.context import ReaderIOResultE
 
-  @final
```

Comment:
I guess this is not a typo. `@final` protocols are ones that cannot be subclasses by other protocols.

Or am I missing something? 🤔

Reply:
Referring to https://github.com/python/mypy/issues/8316:

```python
from typing import Protocol, Callable, Any, final


@final
class User(Protocol):
    fetch: Callable[[int], dict[str,Any]]
# error: Final class t.User has abstract attributes "fetch"
```

Reply:
Thanks! 👍